event.go

```diff
@@ -336,15 +481,26 @@ func (e *Event) SetUnsignedField(path string, value interface{}) error {
 	unsigned := RawJSONFromResult(res, eventJSON)
 
 	e.eventJSON = eventJSON
-	e.fields.Unsigned = unsigned
+	switch fields := e.fields.(type) {
+	case eventFormatV1Fields:
+		fields.Unsigned = unsigned
+		fields.fixNilSlices()
+		e.fields = fields
+	case eventFormatV2Fields:
+		fields.Unsigned = unsigned
+		fields.fixNilSlices()
+		e.fields = fields
```

Comment:
These 2 cases are identical.... use `fallthrough`

Reply:
Not possible with type switches.